### DISCUSSION TITLE
Poll Interface

### DISCUSSION THREAD

**Discussion by Cyklan** [directus/directus#2792] [Feature Requests]
It would be great if there was an out of the box field for creating a poll.
E.g.

#### Poll Title
* Poll option 1
* Poll option 2
* new poll option

Basically a text input with a nested key/value pair to it.

**Comment by rijkvanzanten** [comment]
Wouldn't this already be possible by using the existing repeater or key/value interfaces?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
